docs/release-notes/8.9.asciidoc

```diff
@@ -1,6 +1,30 @@
 [[release-notes-header-8.9.0]]
 == 8.9
 
+[discrete]
+[[release-notes-8.9.1]]
+=== 8.9.1
+
+[discrete]
+[[breaking-changes-8.9.1]]
+==== Breaking changes
+
+There are no breaking changes in 8.9.1.
+
+[discrete]
+[[enhancements-8.9.1]]
+==== Enhancements
+* Event correlation queries and rules can now detect missing events in EQL sequences.
```

Comment:
@nastasha-solomon if Paul doesn't suggest a PR for us to link to, what do you think about including a cross-ref to our create EQL rule doc link here?

Reply:
Yeah, good idea. I'll merge this change tomorrow so the link resolves and the CI checks pass:

```suggestion
* Event correlation queries and rules can now detect {ref}/eql-syntax.html#eql-missing-events[missing events] in EQL sequences.
```

Reply:
As discussed with @chriscressman over slack, I'll remove this ref for now and then re-add it tomorrow in a separate PR.

Reply:
PR for change: https://github.com/elastic/security-docs/pull/3736